### PR TITLE
Create ui-c8y-1018-503-97-clear-cookies-before-relogin-to-the-platform-after-changing-login-mode.md

### PR DESCRIPTION
**Pull request by gen-App-Enablement**
Release note created by amio

Ticket: [MTM-59531](https://cumulocity.atlassian.net/browse/MTM-59531)
Original PR: [6509](https://github.softwareag.com/IOTA/cumulocity-ui/pull/6509)

### CODE REVIEW

File: content/change-logs/platform-services/ui-c8y-1018-503-97-clear-cookies-before-relogin-to-the-platform-after-changing-login-mode.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: clear cookies before relogin to the platform after changing login mode (#6509)
```
Comment (gen-App-Enablement) [inline content/change-logs/platform-services/ui-c8y-1018-503-97-clear-cookies-before-relogin-to-the-platform-after-changing-login-mode.md:3]: The Claude AI suggested to use the following title:
```suggestion
title: Clear cookies before relogin after changing login mode
```

File: content/change-logs/platform-services/ui-c8y-1018-503-97-clear-cookies-before-relogin-to-the-platform-after-changing-login-mode.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-0UgqXH1Ys
+    label: Administration
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-59531
+version: 1018.503.97
+---
+clear cookies before relogin to the platform after changing login mode (#6509)
```
Comment (gen-App-Enablement) [inline content/change-logs/platform-services/ui-c8y-1018-503-97-clear-cookies-before-relogin-to-the-platform-after-changing-login-mode.md:17]: The Claude AI suggested to use the following body:
```suggestion
Previously, after changing the login mode, users were not required to clear their cookies before relogging into the platform, which could lead to unexpected behavior. With this change, users are now prompted to clear their cookies before relogging into the platform after changing the login mode. This ensures a clean session and prevents potential issues related to stale or inconsistent cookie data.
```

**Review by gen-App-Enablement — COMMENTED** [review]
AI improvements

**Comment by kalavathig1** [comment]
This is a test fix, we dont need release notes

**Pull request closed without merging**